client/src/app/core/ui-services/tree.service.ts

```diff
@@ -2,6 +2,7 @@ import { Injectable } from '@angular/core';
 
 import { Displayable } from 'app/site/base/displayable';
 import { Identifiable } from 'app/shared/models/base/identifiable';
+import { OSTreeSortEvent } from 'app/shared/components/sorting-tree/sorting-tree.component';
```

Comment:
Do not import things other then from the core itself. Also the Event has nothing to do with the trees seen here.